staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/versioned/conversion_test.go

```diff
@@ -130,90 +130,3 @@ func TestConvertToGVK(t *testing.T) {
 		})
 	}
 }
-
-func TestConvert(t *testing.T) {
```

Comment:
I specifically asked for this test.  Why is it being removed?

Reply:
There wasn't anything left than a scheme.Convert call. Why do we wrap it in a Convertor struct?

Reply:
would prefer to have a unit test checking for the resulting object after mutation than unit testing our conversion machinery again.

Reply:
Readded the test as a mutating webhook Dispatch unit test.